src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
+import { CommentInfoProps } from '../../types/CommentInfoProps';
 
 import './CommentInfo.scss';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+export const CommentInfo: React.FC<CommentInfoProps> = ({ comment }) => (
+  <>
+    <p className="comment__title" data-cy="comment-name">
+      {comment.name[0].toUpperCase() + comment.name.slice(1)}
+    </p>
+    <p className="comment__body" data-cy="comment-body">
+      {comment.body[0].toUpperCase() + comment.body.slice(1)}
+    </p>
```

Comment:
mb `text-transform:capitalize;` in styles?

Reply:
@Bohdan-Maksymiuk now I checked it precisely and actually `text-transform: capitalize;` capitalizes not the first letter as I need here, but each first letter in the text. So I'm getting back to slice